doc/auth.md

```diff
@@ -230,6 +230,194 @@ Response Payload :
 ### Endpoint
 POST /forgot-password
 
+For recycle your password, you need to input your username and email for sending an password recovery to your email.
+
```

Comment:
You need create new table forgot password for store the link of forgot password request. Please show the sql script below.